### DISCUSSION TITLE
Feature Request: MockBuilder strict mode with standalone components

### DISCUSSION THREAD

**Discussion by StavNoyAkur8** [help-me-mom/ng-mocks#7577] [Ideas]
[Strict mode](https://ng-mocks.sudo.eu/api/MockBuilder#strict-mode) is a great feature.

It is activated when passing 2+ parameters - typically the tested component and it's module.
But standalone components don't have a module to pass as a second parameter.

We can simply pass `null` as the second parameters, or an empty array (which would usually be used for keeping multiple modules), but that's not obvious, and hard to enforce on a large project:
```ts
MockBuilder(TargetComponent, null);
MockBuilder(TargetComponent, []);
```

What I'd like: Detect that a component is standalone, make it activate strict mode.

Alternative: Allow global config of strict mode the default, regardless of standalone. Probably a bigger feature altogether

**Comment by satanTime** [comment]
Thank you for the idea, I've create an FR to implement this: https://github.com/help-me-mom/ng-mocks/issues/7800 feel free to follow there.
